index.html

```diff
@@ -259,6 +270,134 @@ <h3>Video Content Hints</h3>
       </p></td></tr>
     </tbody></table>
   </section>
+  <section>
+    <h2>Behaviors of other components based on content-hint</h2>
+    <section>
+      <h3>Behavior of a MediaStreamTrack</h3>
+      <p>When setting a contentHint value for a MediaStreamTrack, the
+        UA MUST <a>apply a default</a> as follows:
+        <ul>
+          <li>For an audio track with the value "music", and for constraints
+            echoCancellation, autoGainControl and noiseSuppression
+            apply a default of "false".</li>
+          <li>For an audio track with the value "speech", and for constraints
+            echoCancellation and autoGainControl, apply a default of
+            "true".</li>
+        </ul>
+      </p>
+      <p>To <dfn>apply a default</dfn> to the constraint <var>c</var>
+        with value <var>t</var>, perform the following steps:
+        <ul>
+          <li>If the value <var>t</var> satisfies the applied constraints,
+            set the setting corresponding to c <var>t</var>.</li>
+          <li>Otherwise, choose a value for the setting corresponding to c
+            that satisfies the applied constraints.</li>
+          <li>Remember the value <var>t</var>.
+          <li>In parallel, update the trac with the new settings.</li>
+        </ul>
+      <p>Whenever the "apply constraints" algorithm is subsequently
```

Comment:
We probably want to link to the [SelectSettings](https://w3c.github.io/mediacapture-main/getusermedia.html#dfn-selectsettings) algorithm.